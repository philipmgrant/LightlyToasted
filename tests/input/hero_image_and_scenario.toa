|Scenario: incomingcall
|ImageSource: Z:\images\hero.png
|ImagePlacement: hero
testing
incoming call
and hero