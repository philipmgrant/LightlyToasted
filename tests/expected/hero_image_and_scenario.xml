<toast scenario="incomingCall"><visual><binding template="ToastGeneric"><image id="1" src="Z:\images\hero.png" placement="hero"/><text id="1">testing</text><text id="2">incoming call</text><text id="3">and hero</text></binding></visual></toast>